scripts/fornax1.sh

```diff
@@ -0,0 +1,75 @@
+#! /bin/bash
+
+set -x
```

Comment:
if some part of this script fails, do we want the script to continue? if not, add this here

```bash
set -e
```